Dockerfile

```diff
@@ -125,5 +125,5 @@ COPY --from=assets-build /node/dist      /var/www/html/public/app/themes/justice
 
 # Only take what Nginx needs (current configuration)
 COPY --from=build-fpm-composer --chown=www-data:www-data /var/www/html/vendor-assets /var/www/html/
-COPY --from=build-fpm-composer --chown=www-data:www-data /var/www/html/public/index.php /var/www/html/public/index.php
+COPY --from=build-fpm-composer --chown=www-data:www-data /var/www/html/public/wp/index.php /var/www/html/public/index.php
```

Comment:
This looks like a probable typo. 
If we don't have an index.php file in the root of the service Nginx will fail.

Reply:
Ok, just saw that we're picking the file from the WP core. We have a modified file in public here, do we not need this anymore?

Reply:
Sorry, my bad, I was too hasty changing this. I thought it was the same as `public/index.php`. I've reverted this.